kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -838,18 +837,6 @@ public RestMetricsContext getMetricsContext() {
     return metricsContext.metricsContext();
   }
 
-  public int consumerPort(String scheme) throws URISyntaxException {
```

Comment:
Do we need to deprecate this?

Reply:
What do you mean? This is an internal method that is not used anymore?